cub/cub/thread/thread_reduce.cuh

```diff
@@ -227,137 +229,121 @@ namespace internal
 // TODO: add Blackwell support
 
 template <typename T, typename ReductionOp>
-struct enable_generic_simd_reduction_traits
-{
-  static constexpr bool value =
-    cub::detail::is_one_of<T, ::cuda::std::int16_t, ::cuda::std::uint16_t>()
-    && cub::detail::
-      is_one_of<ReductionOp, ::cuda::minimum<>, ::cuda::minimum<T>, ::cuda::maximum<>, ::cuda::maximum<T>>();
-};
+inline constexpr bool enable_generic_simd_reduction_traits_v =
```

Comment:
Remark: We have `_CCCL_GLOBAL_CONSTANT` for global constants, but I am not sure if the workaround (adding `__device__` for the device compilation pass) is still necessary.